lecture/models.py

```
@@ -1,26 +1,55 @@
+    order = models.PositiveIntegerField(default=0)
+
     slug = models.SlugField(max_length=255, unique=True, blank=True)
 
+    class Meta:
+        ordering = ['order']
+
     def __str__(self):
-        return self.title
+        return f"{self.course.title} - {self.title}"
 
     def save(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)